apps/next/src/pages/Portfolio/components/OnRampForm/OnRampForm.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Box,
+  ChevronRightIcon,
+  List,
+  ListItemText,
+  Stack,
+  toast,
+  Typography,
+} from '@avalabs/k2-alpine';
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MdAdd, MdSwapHoriz } from 'react-icons/md';
+import { AvaGradient } from './components/AvaGradient';
+import * as Styled from './components/Styled';
+
+const navigateToBuyPage = () => {
+  window.open('https://core.app/buy', '_blank');
```

Comment:
Let's take the URL from `@/constants`:

https://github.com/ava-labs/core-extension/blob/0224880da4ea4acfc0b4752fe8954838c0cf2696/apps/next/src/config/constants.ts#L1-L2